src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/MitochondrialFiltersArgumentCollection.java

```diff
@@ -0,0 +1,26 @@
+package org.broadinstitute.hellbender.tools.walkers.mutect;
+
+import org.broadinstitute.barclay.argparser.Argument;
+
+
+public class MitochondrialFiltersArgumentCollection extends M2FiltersArgumentCollection {
+    private static final long serialVersionUID = 9345L;
+    public static final String TLOD_BY_DEPTH = "tlod-divided-by-depth";
+    public static final String NON_MT_ALT_READS_BY_ALT_READS = "non-mt-alts-divided-by-alts";
+
+    /**
+     * Only variants with TLOD divided by depth exceeding this threshold can pass filtering.
+     */
+    @Argument(fullName = TLOD_BY_DEPTH,
+            doc="TLOD by depth threshold for filtering variant", optional = true)
+    public double tlod_by_depth = .005;
```

Comment:
It would be nice to be able to say what this number means.  For example, there are a few hand wavy explanations running around for the magic 6.3 for Mutect2, like two reads with Q30 alts.  That might be less intuitive for something that's odds-ratio-based rather than probability-based though.